transactions_test.go

```diff
@@ -121,6 +121,7 @@ func TestTransactions_List(t *testing.T) {
 			Voidable:         recurly.NewBool(true),
 			Refundable:       recurly.NewBool(true),
 			IPAddress:        net.ParseIP("127.0.0.1"),
+			TransactionError: &recurly.TransactionError{},
```

Comment:
Are these not coming back `nil`? What happens if you omit this line all together? Ideally the user could have a transaction and see if `transaction.TransactionError` is non-nil. If it's returning back this way, the user will always get a non-nil transaction error but not know if there is actually a transaction error present.